docs/containers.conf.5.md

```diff
@@ -361,6 +361,10 @@ pod, we start a `pause` process in a container to hold open the namespaces
 associated with the  pod.  This container does nothing other then sleep,
 reserving the pods resources for the lifetime of the pod.
 
+**image_parallel_pulls**=6
```

Comment:
I think we need to `s/pull/copy` as it also applies to push or, in theory, any other copy operation between the various transports (as long as they support it).

Reply:
`image_max_copy_ops` ?

Reply:
`image_copy_max_parallel_ops` ?

The description should likewise not talk about pulls only.

Reply:
Perhaps `max_parallel_copies` or `max_parallel_layer_copies`, and then explain it applies to the number of layers pushed or pulled in the verbiage?

Reply:
I'm not real sold on either of my suggestions, just throwing it out for other thoughts.

Reply:
I will change to to copy.  Lets not get too technical.